### DISCUSSION TITLE
Apache vhost as reverse proxy

### DISCUSSION THREAD

**Discussion by cesare190** [goauthentik/authentik#9085] [Q&A]
Hello, i don't find anything similar so i'll try to ask! I'm [reading this page](https://docs.goauthentik.io/docs/providers/proxy/forward_auth) (Forward Auth) but there is nothing about apache2. I know that nginx and similar should be the way to go but in my environment Apache is the WebServer to use. Is it possible to use Apache? If yes, can be published also the default snippet for this webserver?

Thanks so much! I hope I haven't written anything stupid

**Comment by donwong527** [comment]
Hey did you find anything? I am also stuck here now...

**Comment by se4598** [comment]
As far as I know, there is no [out-of-the-box solution](https://httpd.apache.org/docs/2.4/mod/) for apache2 (v2.4).

But there is a 3rd-party module you can install which supports OpenID Connect and can be used: [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc)

You can find an example snipped in their wiki/FAQ.
